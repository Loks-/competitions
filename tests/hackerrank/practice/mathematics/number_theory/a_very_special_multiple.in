8
4
5
80
62
67
20
63
81
