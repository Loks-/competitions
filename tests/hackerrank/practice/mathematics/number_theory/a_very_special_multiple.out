2
3
4
30
66
3
36
162
